.github/workflows/unit_tests_with_latest_deps.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - master
 
-name: Unit Tests - Latest Dependencies
+name: Unit Tests
```

Comment:
Renamed this because it is used for the badge
<img width="311" alt="Screen Shot 2021-05-24 at 2 56 18 PM" src="https://user-images.githubusercontent.com/8726321/119394475-3383d880-bca0-11eb-9ac9-63480bfd86ed.png">